src/main/java/com/epam/tc/hw3/library/pages/Header.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.tc.hw3.library.pages;
+
+import com.epam.tc.hw3.library.utils.GetProperties;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Header {
+
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+    @FindBy(id = "name")
+    private WebElement usernameField;
+    @FindBy(id = "password")
+    private WebElement passwordField;
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+
+    @FindBy(id = "user-name")
+    private WebElement userNameElement;
+
+    @FindBy(css = "a[href='index.html']")
+    private WebElement homeButton;
+    @FindBy(css = "a[href='contacts.html']")
+    private WebElement contactFormButton;
+    @FindBy(css = "a.dropdown-toggle")
+    private WebElement serviceButton;
+    @FindBy(css = "a[href='metals-colors.html']")
+    private WebElement metalsColorsButton;
+
+
+    //Items in Service drop down:
+    @FindBy(css = "a[href='different-elements.html']")
+    private WebElement difElements;
```

Comment:
It's really hard to read. Add blank lines between them

Reply:
Fixed